docs/tutorial.rst

```diff
@@ -331,61 +399,20 @@ that will actually do the conversion:
        return dst_dt
 
 
-And below is a simple ``show_time.html`` template.
-Note how the ``text`` and ``value`` subitems are used:
-
-.. code-block:: html
-
-   <!DOCTYPE html>
-   <html lang="en">
-   <head>
-     <meta charset="utf-8">
-     <title>Time zone convertor</title>
-   </head>
-   <body>
-     <h1>Time zone convertor</h1>
-     <p class="info">
-       When it's <time datetime="{src_dt.value}">{src_dt.text}</time>
-       in {src_location},<br>
-       it's <time datetime="{dst_dt.value}">{dst_dt.text}</time>
-       in {dst_location}.
-     </p>
-     <p>Go to the <a href="/">home page</a>.</p>
-   </body>
-   </html>
-
-
 Static assets
 -------------
 
-As our next step, let us apply some style to our markup.
+As our next step, let us apply some style to our page.
```

Comment:
```suggestion
As our next step, let us apply some style to our page.
Adding styles will require a CSS file, which should be served without processing.
Such files are generally known as *static assets*.
```
I feel like this explains the term "static" and the fact that it's a general term, not specific to Clastic.